### PR TITLE
Events Detail View 

### PR DESCRIPTION
**Pull request by pandilladivya**
@sandarumk @HimanshiJain 
Kindly review the PR.
Thanks.

### CODE REVIEW

File: app/src/main/java/vola/systers/com/android/activities/EventDetailView.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package vola.systers.com.android.activities;
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/EventDetailView.java]: Maintain consistency as @sandarumk also suggested. Should not it be EventDetailViewActivity?

File: app/src/main/java/vola/systers/com/android/activities/LocationDetails.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package vola.systers.com.android.activities;
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: Same here for the name of the activity.

Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: By the way where are we using this activity and what does it actually do?

File: app/src/main/res/layout/activity_event_detail_view.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="vola.systers.com.android.activities.EventDetailView">
+
+    <android.support.design.widget.AppBarLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:theme="@style/AppTheme.AppBarOverlay">
+
+        <android.support.v7.widget.Toolbar
```
Comment (HimanshiJain) [inline app/src/main/res/layout/activity_event_detail_view.xml]: Would it be good to have a back icon on the left most part of the Toolbar on this activity? And shouldn't the toolbar display the name of the event?

File: app/src/main/java/vola/systers/com/android/activities/LocationDetails.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        if(url!=null)
+            wv1.loadUrl(url);
+        else
+            wv1.setVisibility(View.GONE);
+
+    }
+
+    private class MyBrowser extends WebViewClient {
+        @Override
+        public boolean shouldOverrideUrlLoading(WebView view, String url) {
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: This is deprecated right? Should we use it?

Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: Also replace other deprecated functions if used elsewhere.

**Comment by HimanshiJain** [comment]
And resolve conflicts.

File: app/src/main/java/vola/systers/com/android/activities/LocationDetails.java (hunk 2 above)
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: Yeah. we are not using. I will remove.

File: app/src/main/java/vola/systers/com/android/activities/LocationDetails.java (hunk 4 above)
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/activities/LocationDetails.java]: Okay. And I am not using this Class. I will remove this. :)

File: app/src/main/java/vola/systers/com/android/activities/EventDetailView.java (hunk 1 above)
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/activities/EventDetailView.java]: Okay.

File: app/src/main/res/layout/activity_event_detail_view.xml (hunk 3 above)
Comment (pandilladivya) [inline app/src/main/res/layout/activity_event_detail_view.xml]: Agreed. Will change

**Comment by pandilladivya** [comment]
@HimanshiJain @sandarumk 
Implemented all suggested changes. Also resolved conflicts.
Kindly review.

File: app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -55,23 +56,30 @@
 
         <activity android:name="vola.systers.com.android.activities.MenuActivity">
             <intent-filter>
-                <action android:name="android.intent.action.MAIN"
-                    android:theme="@style/AppTheme.NoActionBar"/>
+                <action android:name="android.intent.action.MAIN" />
 
-                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.LAUNCHER" />
```
Comment (HimanshiJain) [inline app/src/main/AndroidManifest.xml]: Why this change? This means that the first time the app is installed, the MenuActivity will be launched . When will the user login?

File: app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -55,23 +56,30 @@
...
 
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+
         <activity android:name="vola.systers.com.android.activities.SignInActivity">
                 <intent-filter>
                     <action android:name="android.intent.action.MAIN" />
-                    <category android:name="android.intent.category.LAUNCHER" />
+                    <category android:name="android.intent.category.DEFAULT" />
```
Comment (HimanshiJain) [inline app/src/main/AndroidManifest.xml]: Same as above?

Comment (pandilladivya) [inline app/src/main/AndroidManifest.xml]: Sorry. Will modify. I put this for testing and forgot to rechange.

Comment (pandilladivya) [inline app/src/main/AndroidManifest.xml]: @HimanshiJain Done with changing and amended the changes to my previous commit :)
